source/opt/debug_info_manager.h

```diff
@@ -107,7 +93,9 @@ class DebugInfoManager {
   void RegisterDbgInst(Instruction* inst);
 
   // Registers the DebugFunction |inst| into |fn_id_to_dbg_fn_| using the
-  // function operand of |inst| as a key.
+  // function operand of |inst| as a key. If the function operand of |inst| is
+  // already registered with a different DebugFunction, this function will
+  // result in an assertion failure.
```

Comment:
You should not be talking about the internals of the class in the external interface.  You can simply say "Register the DebugFunction instruction |inst|.  The function referenced in |inst| must not already be registered."